include/lbann/callbacks/callback_learning_rate.hpp

```diff
@@ -229,6 +229,42 @@ class lbann_callback_linear_growth_learning_rate :
   int64_t m_delay;
 };
 
+/**
+ * Decrease the learning rate by polynomial policy
+ * base_lr*(1 - i_cur/i_max)^p, where
+ * base_lr is the initial learning rate, i_cur is the current iteration,
+ * i_max is the maximum iteration, and p is a parameter.
+ */
+class lbann_callback_poly_learning_rate : public lbann_callback_learning_rate {
+ public:
+  lbann_callback_poly_learning_rate(double p, uint64_t n_epochs, uint64_t max_iter);
```

Comment:
We should document that `n_epochs` is not used when `max_iter` is non-zero.

Reply:
Adding some comments about it.